### ISSUE TITLE
Unifi Protect Application Reboot

### ISSUE THREAD

**Issue by albert-gharibi** [hjdhjd/homebridge-unifi-protect#1047]
### Homebridge UniFi Protect Version

6.18.1

### Homebridge Platform and OS

Raspberry Pi

### Homebridge Version

1.7.0

### Node Version

20.11.0

### UniFi OS Version

3.2.12

### UniFi Protect Controller Version

2.11.21

### Describe the problem

It's been a month that I have noticed my Unifi Protect cameras randomly go offline and then come back online in a few seconds. I realized that the Unifi Protect application is rebooting itself randomly. I contacted the Unifi support and they recommended to disable the Unifi Protect plug in on my homebridge to see if it is the cause of the issue. I disabled the protect plug in on homebridge and the issue has gone away. Looks like there is something with the plug in that causes the Unifi Protect application to reboot itself randomly. Can you please look into this issue?

### Homebridge HBUP JSON configuration

```json
{
    "bridge": {
        "name": "Homebridge CD18",
        "username": "0E:03:4E:BA:3B:72",
        "port": 51758,
        "pin": "923-65-365",
        "advertiser": "avahi",
        "bind": [
            "eth0"
        ]
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "dark-mode",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "controllers": [
                {
                    "address": "192.168.10.1",
                    "name": "Jarvis",
                    "password": "Homebridge2022@",
                    "username": "homebridge"
                }
            ],
            "options": [
                "Enable.Motion.SmartDetect",
                "Enable.Motion.SmartDetect.68D79AE57601"
            ],
            "verboseFfmpeg": false,
            "_bridge": {
                "username": "0E:C3:04:9F:38:54",
                "port": 54855
            },
            "platform": "UniFi Protect"
        }
    ],
    "disabledPlugins": [
        "homebridge-unifi-protect"
    ]
}
```

### Relevant log output

```shell
[25/02/2024, 21:59:00] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Enabled doorbell message switch: DO NOT DISTURB.
[26/02/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[26/02/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[26/02/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-1206Hs/homebridge-backup-0E034EBA3B72.tar.gz
[26/02/2024, 14:24:55] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Doorbell ring detected.
[26/02/2024, 14:24:55] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.180: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[26/02/2024, 14:24:57] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.181: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[26/02/2024, 14:25:27] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[26/02/2024, 14:25:29] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[26/02/2024, 14:28:50] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Right [G4 Bullet]: Unable to retrieve a snapshot.
[26/02/2024, 14:28:50] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Door [G4 Bullet]: Unable to retrieve a snapshot.
[26/02/2024, 14:28:51] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Left [G4 Bullet]: Unable to retrieve a snapshot.
[26/02/2024, 14:28:51] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Left [G4 Bullet]: Unable to retrieve a snapshot.
[26/02/2024, 14:28:51] [homebridge-unifi-protect] 192.168.10.1 G4 Doorbell [G4 Doorbell]: Unable to open the return audio channel.
[26/02/2024, 14:28:53] [homebridge-unifi-protect] 192.168.10.1 G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.180 (high latency connection): 640x480@30fps, 164 kbps. Transcoding 960x720@30fps (Medium), 1,200 kbps.
[26/02/2024, 14:28:55] [homebridge-unifi-protect] 192.168.10.1 G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[27/02/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[27/02/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[27/02/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-gOBYKu/homebridge-backup-0E034EBA3B72.tar.gz
[28/02/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[28/02/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[28/02/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-XUvOax/homebridge-backup-0E034EBA3B72.tar.gz
[28/02/2024, 15:24:26] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Doorbell ring detected.
[28/02/2024, 15:24:27] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.180: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[28/02/2024, 15:24:28] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.181: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[28/02/2024, 15:24:40] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Doorbell ring detected.
[28/02/2024, 15:24:58] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[28/02/2024, 15:25:00] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[28/02/2024, 15:31:58] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Doorbell ring detected.
[28/02/2024, 15:31:59] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.180: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[28/02/2024, 15:32:00] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.181: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[28/02/2024, 15:32:30] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[28/02/2024, 15:32:31] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[28/02/2024, 15:51:26] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Doorbell ring detected.
[28/02/2024, 15:51:26] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.180: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[28/02/2024, 15:51:27] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Streaming request from 192.168.10.181: 1600x1200@30fps, 1,450 kbps. Using 1600x1200@30fps (High), 6,000 kbps.
[28/02/2024, 15:51:57] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[28/02/2024, 15:51:59] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Stopped video streaming session.
[29/02/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[29/02/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[29/02/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-dpg3fw/homebridge-backup-0E034EBA3B72.tar.gz
[01/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[01/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[01/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-zfoQcw/homebridge-backup-0E034EBA3B72.tar.gz
[02/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[02/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[02/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-HwJSBu/homebridge-backup-0E034EBA3B72.tar.gz
[03/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[03/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[03/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-ev4eBu/homebridge-backup-0E034EBA3B72.tar.gz
[04/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[04/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[04/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-VPRT4w/homebridge-backup-0E034EBA3B72.tar.gz
[05/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[05/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[05/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-RuMVfu/homebridge-backup-0E034EBA3B72.tar.gz
[05/03/2024, 21:03:12] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Right [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:12] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Left [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:12] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Door [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:12] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Left [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:13] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Left [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:13] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Door [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:13] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Left [G4 Bullet]: Unable to retrieve a snapshot.
[05/03/2024, 21:03:13] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Right [G4 Bullet]: Unable to retrieve a snapshot.
[06/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[06/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[06/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-qgiHXw/homebridge-backup-0E034EBA3B72.tar.gz
[07/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[07/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[07/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-Do137u/homebridge-backup-0E034EBA3B72.tar.gz
[07/03/2024, 14:23:11] [Homebridge UI] Running Command: /opt/homebridge/lib/node_modules/homebridge-config-ui-x/plugin-upgrade-install.sh homebridge-unifi-protect 6.18.1 /var/lib/homebridge/node_modules
[07/03/2024, 14:23:29] [Homebridge UI] Homebridge restart request received
[07/03/2024, 14:23:29] [Homebridge UI] UI / Bridge settings have not changed; only restarting Homebridge process
[07/03/2024, 14:23:29] [Homebridge UI] Sending SIGTERM to Homebridge
[07/03/2024, 14:23:29] Got SIGTERM, shutting down Homebridge...
[07/03/2024, 14:23:29] Got SIGTERM, shutting down child bridge process...
[07/03/2024, 14:23:34] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[07/03/2024, 14:23:39] [HB Supervisor] Restarting Homebridge...
[07/03/2024, 14:23:39] [HB Supervisor] Starting Homebridge with extra flags: -I -P /var/lib/homebridge/node_modules --strict-plugin-resolution
[07/03/2024, 14:23:39] [HB Supervisor] Started Homebridge v1.7.0 with PID: 25055
[07/03/2024, 14:23:41] Loaded config.json with 0 accessories and 2 platforms.
[07/03/2024, 14:23:42] Loaded 0 cached accessories from cachedAccessories.
[07/03/2024, 14:23:42] ---
[07/03/2024, 14:23:43] Loaded plugin: homebridge-unifi-protect@6.18.1
[07/03/2024, 14:23:43] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[07/03/2024, 14:23:43] ---
[07/03/2024, 14:23:43] Loading 2 platforms...
[07/03/2024, 14:23:43] [UniFi Protect] Initializing UniFi Protect platform...
[07/03/2024, 14:23:43] [UniFi Protect] Initializing child bridge 0E:C3:04:9F:38:54
Setup Payload:
X-HM://0024ZX9O52PQ1
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 923-65-365 │     
    └────────────┘     
                       
[07/03/2024, 14:23:43] Homebridge v1.7.0 (HAP v0.11.1) (Homebridge CD18) is running on port 51758.
[07/03/2024, 14:23:45] [homebridge-unifi-protect] Launched child bridge with PID 25086
[07/03/2024, 14:23:46] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[07/03/2024, 14:23:46] [homebridge-unifi-protect] Loaded homebridge-unifi-protect v6.18.1 child bridge successfully
[07/03/2024, 14:23:46] Loaded 7 cached accessories from cachedAccessories.0EC3049F3854.
[07/03/2024, 14:23:47] [homebridge-unifi-protect] Using FFmpeg version: 6.0-homebridge-alpine-armv7l-static.
[07/03/2024, 14:23:47] Homebridge v1.7.0 (HAP v0.11.1) (homebridge-unifi-protect) is running on port 54855.
(node:25086) [DEP0123] DeprecationWarning: Setting the TLS ServerName to an IP address is not permitted by RFC 6066. This will be ignored in a future version.
(Use `node --trace-deprecation ...` to show where the warning was created)
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Connected to 192.168.10.1 (UniFi Protect 2.11.21 running on UniFi OS 3.2.12).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered nvr: Jarvis [UDM Pro] (address: 192.168.10.1 mac: 74ACB93B6E63).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: Front Door [G4 Bullet] (address: 192.168.50.229 mac: 68D79AE57601).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: Back Right [G4 Bullet] (address: 192.168.50.160 mac: 68D79AE49D06).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: Front Right [G4 Bullet] (address: 192.168.50.154 mac: 68D79AE49CBD).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: Back Side [G4 Bullet] (address: 192.168.50.122 mac: 68D79AE57631).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: Back Left [G4 Bullet] (address: 192.168.50.144 mac: 68D79AE49CB3).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: Front Left [G4 Bullet] (address: 192.168.50.189 mac: 68D79AE49C45).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO]: Discovered camera: G4 Doorbell [G4 Doorbell] (address: 192.168.20.251 mac: 68D79AE583C9).
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Door [G4 Bullet]: Smart motion detection enabled: animal, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Door [G4 Bullet]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Right [G4 Bullet]: Smart motion detection enabled: animal, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Right [G4 Bullet]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Right [G4 Bullet]: Smart motion detection enabled: animal, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Right [G4 Bullet]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Side [G4 Bullet]: Smart motion detection enabled: animal, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Side [G4 Bullet]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Left [G4 Bullet]: Smart motion detection enabled: animal, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Back Left [G4 Bullet]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Left [G4 Bullet]: Smart motion detection enabled: animal, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] Front Left [G4 Bullet]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Smart motion detection enabled: animal, package, person, vehicle.
[07/03/2024, 14:23:48] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: WARNING: Smart motion detection and HomeKit Secure Video provide overlapping functionality. Only HomeKit Secure Video, when event recording is enabled in the Home app, will be used to trigger motion event notifications for this camera.
[07/03/2024, 14:23:49] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Enabled doorbell message switch: LEAVE PACKAGE AT DOOR.
[07/03/2024, 14:23:49] [homebridge-unifi-protect] Jarvis [UDM-PRO] G4 Doorbell [G4 Doorbell]: Enabled doorbell message switch: DO NOT DISTURB.
[07/03/2024, 16:35:41] [Homebridge UI] Changes to config.json saved.
[07/03/2024, 16:35:41] [homebridge-unifi-protect] Stopping child bridge (will not restart)...
[07/03/2024, 16:35:41] Got SIGTERM, shutting down child bridge process...
[07/03/2024, 16:35:43] [Homebridge UI] Homebridge restart request received
[07/03/2024, 16:35:43] [Homebridge UI] UI / Bridge settings have not changed; only restarting Homebridge process
[07/03/2024, 16:35:43] [Homebridge UI] Sending SIGTERM to Homebridge
[07/03/2024, 16:35:43] Got SIGTERM, shutting down Homebridge...
[07/03/2024, 16:35:46] [homebridge-unifi-protect] Child bridge process ended
[07/03/2024, 16:35:48] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[07/03/2024, 16:35:53] [HB Supervisor] Restarting Homebridge...
[07/03/2024, 16:35:53] [HB Supervisor] Starting Homebridge with extra flags: -I -P /var/lib/homebridge/node_modules --strict-plugin-resolution
[07/03/2024, 16:35:53] [HB Supervisor] Started Homebridge v1.7.0 with PID: 28613
[07/03/2024, 16:35:55] Loaded config.json with 0 accessories and 2 platforms.
[07/03/2024, 16:35:55] Loaded 0 cached accessories from cachedAccessories.
[07/03/2024, 16:35:55] ---
[07/03/2024, 16:35:56] Disabled plugin: homebridge-unifi-protect@6.18.1
[07/03/2024, 16:35:56] ---
[07/03/2024, 16:35:56] Loading 2 platforms...
[07/03/2024, 16:35:56] Ignoring config for the platform "UniFi Protect" in your config.json as the plugin "homebridge-unifi-protect" has been disabled.
Setup Payload:
X-HM://0024ZX9O52PQ1
Enter this code with your HomeKit app on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 923-65-365 │     
    └────────────┘     
                       
[07/03/2024, 16:35:56] Homebridge v1.7.0 (HAP v0.11.1) (Homebridge CD18) is running on port 51758.
[08/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[08/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[08/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-3tpeEt/homebridge-backup-0E034EBA3B72.tar.gz
[09/03/2024, 01:10:05] [Homebridge UI] Running job to cleanup config.json backup files older than 60 days...
[09/03/2024, 04:41:36] [Homebridge UI] Running scheduled instance backup...
[09/03/2024, 04:41:36] [Homebridge UI] Creating temporary backup archive at /tmp/homebridge-backup-eyntYv/homebridge-backup-0E034EBA3B72.tar.gz
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Appreciate the bug report, but unfortunately there's not much here for me to specifically troubleshoot.

If you have a specific, reproducible problem, please open a new issue with the steps to reproduce and I'll be happy to look into it. That said, here's a couple of observations based on what you described:

1. You're running HBUP on a low-power/capability platform like Raspberry Pi. Yes, the Pi 4 will work, but it's still a low-power platform that isn't going to scale up beyond a couple of cameras, especially if you use HKSV with HBUP.
2. You're running Protect on your UDMP. Yes, this too will work, but when using HBUP with HKSV, it's going to open persistent video connections to the Protect controller, which in this case you are also using as your primary network router. To exacerbate matters, you're doing it on a low-power platform that's going to struggle to consume all those video streams at once.

TL;DR: if you want to use HKSV, understand you're intentionally selecting a more resource-intensive set of capabilities and follow the HKSV best practices laid out in the documentation, chief among them being to ensure you have robust hardware that you're running HBUP on.

Best of luck...and, beyond the documentation, feel free to ask more performance optimization questions on the Homebridge Discord.

**Comment by albert-gharibi** [comment]
Thanks for the quick response. I am using the HomeKit route for my UniFi cameras mostly for the notification purposes that displays on my Apple TV. Is there a way to disable the HKSV feature to reduce the resource consumption level if that is an option?Thanks,On Mar 9, 2024, at 4:25 PM, hjdhjd ***@***.***> wrote:﻿
Appreciate the bug report, but unfortunately there's not much here for me to specifically troubleshoot.
If you have a specific, reproducible problem, please open a new issue with the steps to reproduce and I'll be happy to look into it. That said, here's a couple of observations based on what you described:

You're running HBUP on a low-power/capability platform like Raspberry Pi. Yes, the Pi 4 will work, but it's still a low-power platform that isn't going to scale up beyond a couple of cameras, especially if you use HKSV with HBUP.
You're running Protect on your UDMP. Yes, this too will work, but when using HBUP with HKSV, it's going to open persistent video connections to the Protect controller, which in this case you are also using as your primary network router. To exacerbate matters, you're doing it on a low-power platform that's going to struggle to consume all those video streams at once.

TL;DR: if you want to use HKSV, understand you're intentionally selecting a more resource-intensive set of capabilities and follow the HKSV best practices laid out in the documentation, chief among them being to ensure you have robust hardware that you're running HBUP on.
Best of luck...and, beyond the documentation, feel free to ask more performance optimization questions on the Homebridge Discord.

**Comment by hjdhjd** [comment]
To do so, don't enable HKSV within the Home app for your Protect cameras.. Beyond that, I refer you to the documentation I pointed to above, as well as googling around for how to enable/disable HKSV within the Home app...MacRumors, etc all have plenty of guides.

Final point: the Protect controller shouldn't be randomly rebooting regardless...and doesn't for most of us. Correlation isn't necessarily causation...I'm deeply skeptical, but open, to HBUP being at fault. Largely because I simply don't hear reports that it does and HBUP has been around quite a long time and used by a large user population. If it were a widespread issue, I would expect it to surface and I haven't seen that.

**Comment by albert-gharibi** [comment]
I understand. The problem is that I have had this setup for almost two years and never had an issue but for this last month after some updates on both UniFi protect and the home bridge and the plug in itself. That is why it led me to think the updates may have caused this issue. I really appreciate your guidance and recommendation.Thanks,On Mar 9, 2024, at 4:57 PM, hjdhjd ***@***.***> wrote:﻿
To do so, don't enable HKSV within the Home app for your Protect cameras.. Beyond that, I refer you to the documentation I pointed to above, as well as googling around for how to enable/disable HKSV within the Home app...MacRumors, etc all have plenty of guides.
Final point: the Protect controller shouldn't be randomly rebooting regardless...and doesn't for most of us. Correlation isn't necessarily causation...I'm deeply skeptical, but open, to HBUP being at fault. Largely because I simply don't hear reports that it does and HBUP has been around quite a long time and used by a large user population. If it were a widespread issue, I would expect it to surface and I haven't seen that.

**Comment by hjdhjd** [comment]
Appreciated - you can always revert to prior versions of HBUP to test further if you like...it's easy to do in the Homebridge UI. Generally speaking, Ubiquiti updates are typically the culprit but I know that's not of much help I'm afraid.

Best of luck!
